packages/eslint-config-vivy/README.md

```diff
@@ -0,0 +1,45 @@
+# eslint-config-vivy
+
+A Vivy eslint configuration ready to be used in multiple projects.
+
+## Installation
+```
+npm install --save-dev eslint eslint-config-vivy
+```
+
+## Usage
+
+Once the `eslint-config-vivy` package is installed, you can use it by specifying `vivy` in the [`extends`](http://eslint.org/docs/user-guide/configuring#extending-configuration-files) section of your [ESLint configuration](http://eslint.org/docs/user-guide/configuring).
+
+```json
+{
+  "extends": "vivy",
```

Comment:
Are you sure this works? How do you define that the eslint config is called "vivy" I also can't see easily how airbnb is doing it

Reply:
yeah i tested it on some projects. I think eslint has this as prefix by default.

Reply:
yeah must be, I love magic 🙄